spec/models/admin_menu_spec.rb

```diff
@@ -88,17 +88,13 @@
     let(:customization) { described_class.navigation_items.fetch(:customization) }
 
     it { is_expected.to be_a(Menu::Item) }
-
-    context "when :profile_field FeatureFlag is enabled" do
-      before { allow(FeatureFlag).to receive(:enabled?).with(:profile_admin).and_return(true) }
-
-      it { is_expected.to be_visible }
-    end
+    it { is_expected.to be_visible }
 
     context "when :profile_field FeatureFlag is not enabled" do
+      # leaving this in place to make sure item is visible even if flag not enabled
```

Comment:
probably want to remove this test case completely when the feature flag is removed.